kodilla-patterns/src/main/java/com/kodilla/patterns/builder/checkers/Figure.java

```diff
@@ -0,0 +1,25 @@
+package com.kodilla.patterns.builder.checkers;
+
+public class Figure
+{
+    public static final String WHITE = "WHITE";
+    public static final String BLACK = "BLACK";
+    final private String color;
+
+    public Figure(String color)
+    {
+        if (color.equals(WHITE) || color.equals(BLACK))
```

Comment:
jak napiszesz WHITE.equals(color) to uodpornisz swój kod na NullPointerException

Reply:
Wyjaśnione na rozmowie - kod z kursu.